docs/guides/guide-using-victoriametrics-to-monitor-vmware-vcenter.md

```diff
@@ -0,0 +1,617 @@
+    </tr>
+    <tr>
+      <td colspan="2" align="center"><strong>Installation</strong></td>
+    </tr>
+    <tr>
+      <td>Binary file with understandable configuration files.</td>
+      <td>Installation by python pip package manager  or docker with configuration example from readme.</td>
+    </tr>
+  </tbody>
+</table>
+
+## Performance counters
+
+Through the UI in vCenter you can see different graphs for different counters. Sometimes it is not clear which one you need to poll. There is [govmomi](https://github.com/vmware/govmomi) for that. To see them
+
+```bash
+cd /tmp
+wget https://github.com/vmware/govmomi/releases/download/v0.27.2/govc_Linux_x86_64.tar.gz
+tar zxvf govc_Linux_x86_64.tar.gz
+```
+
+We need to set this variables to connect to vCenter by running them in terminal:
+
+```bash
+export GOVC_USERNAME=vmuser
+export GOVC_PASSWORD=pswd
+export GOVC_URL=vcenter-host
+export GOVC_INSECURE=true
+```
+
+To configure telegraf we will need to set the correct interval. By default this value is 5 minutes.
+
+Run this command to test it:
+
+```bash
+./govc metric.interval.info
+```
+
+The expected result will look like:
+
+```bash
+ID:                   300
+  Enabled:            true
+  Interval:           5m
+  Available Samples:  288
+  Name:               Past day
+  Level:              1
+ID:                   1800
+  Enabled:            true
+  Interval:           30m
+  Available Samples:  336
+  Name:               Past week
+  Level:              1
+ID:                   7200
+  Enabled:            true
+  Interval:           2h
+  Available Samples:  360
+  Name:               Past month
+  Level:              1
+ID:                   86400
+  Enabled:            true
+  Interval:           24h
+  Available Samples:  365
+  Name:               Past year
+  Level:              1
+```
+
+Run this command to export list of available metrics to file `vm.out`:
+
+```bash
+./govc metric.ls /*/vm/* > vm.out
+```
+
+This command will show information about a specific meter `cpu.usage.average`:
+
+```bash
+./govc metric.info - cpu.usage.average
+```
+
+The result is similar to the example given in the documentation:
+
+```bash
+Name:                cpu.usage.average
+  Label:             Usage
+  Summary:           CPU usage as a percentage during the interval
+  Group:             CPU
+  Unit:              %
+  Rollup type:       average
+  Stats type:        rate
+  Level:             1
+    Intervals:       Past day,Past week,Past month,Past year
+  Per-device level:  3
+    Intervals:
+```
+
+## VictoriaMetrics
+
+It is assumed that VictoriaMetrics(Cluster or Single) and vmagent are installed and running. The agent should be installed on hosts with vmware_exporter, telegraf, so that if the base is unavailable, the received metrics can be stored in the cache on the local disk. Here we need to decide how often metrics will be written to database - for example, every minute. Then minimal config for vmagent is the following:
```

Comment:
Can we link to installation guides?

Reply:
> The agent should be installed on hosts with vmware_exporter, telegraf, so that if the base is unavailable, the received metrics can be stored in the cache on the local disk

It would store metrics disk even if installed on a separate node, wouldn't it?